src/patterns/ec2-app/base.ts

```diff
@@ -158,176 +194,11 @@ function restrictedCidrRanges(ranges: IPeer[]) {
 }
 
 /**
- * Pattern which creates the resources needed to run an application on EC2 instances.
- * For convenience, you may wish to use [[`GuPlayApp`]] or [[`GuNodeApp`]], which extend this class.
- *
- * This pattern will grant your EC2 instances a number of commonly needed IAM permissions. For more information on
- * this, see [[`GuInstanceRole`]]. To add additional permissions to your EC2 instances, see [[`GuEc2AppProps`]].
- *
- * This pattern will automatically create security groups/rules which allow for:
- * 1. Incoming traffic over HTTPS (from the whole internet or from the specified CIDR ranges, depending on the
- * [[`AppAccess`]] specified).
- * 2. Communication between the load balancer and the EC2 instances over HTTP, via the specified application port.
- * 3. Outbound traffic from your EC2 instances over HTTPS (to enable communication with third-party APIs).
- *
- * Example usage for a public facing app (open to the internet):
- * ```typescript
- * new GuEc2App(stack, {
- *   applicationPort: 1234,
- *   app: "app-name",
- *   access: { scope: AccessScope.PUBLIC },
- *   instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.MEDIUM),
- *   certificateProps:{
- *     domainName: "domain-name-for-your-application.example",
- *   },
- *   monitoringConfiguration: {
- *     snsTopicName: "alerts-topic-for-my-team",
- *     http5xxAlarm: {
- *       tolerated5xxPercentage: 1,
- *     },
- *     unhealthyInstancesAlarm: true,
- *   },
- *   userData: {
- *     distributable: {
- *       fileName: "app-name.deb",
- *       executionStatement: `dpkg -i /app-name/app-name.deb`,
- *     }
- *   },
- * });
- * ```
- *
- * Example usage for an app which is restricted to specific CIDR ranges:
- * ```typescript
- * new GuEc2App(stack, {
- *   applicationPort: 1234,
- *   app: "app-name",
- *   access: {
- *     scope: AccessScope.RESTRICTED,
- *     cidrRanges: [Peer.ipv4("192.168.1.1/32"), Peer.ipv4("8.8.8.8/32")],
- *   },
- *   instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.MEDIUM),
- *   certificateProps:{
- *     domainName: "domain-name-for-your-application.example",
- *   },
- *   monitoringConfiguration: {
- *     snsTopicName: "alerts-topic-for-my-team",
- *     http5xxAlarm: {
- *       tolerated5xxPercentage: 1,
- *     },
- *     unhealthyInstancesAlarm: true,
- *   }
- *   userData: {
- *     distributable: {
- *       fileName: "app-name.deb",
- *       executionStatement: `dpkg -i /app-name/app-name.deb`,
- *     }
- *   },
- * });
- * ```
- *
- * ### Varying values by stage
```

Comment:
This is really useful information, but it is not related to the pattern so I wondered if it was better provided elsewhere?